Code/GraphMol/MolInterchange/Writer.cpp

```diff
@@ -393,6 +396,12 @@ void addSubstanceGroup(const SubstanceGroup &sg, rj::Value &rjSG,
   }
 }
 
+double rjPrepareForTrunc(double n) {
```

Comment:
This is complicated enough to deserve a comment, both for purpose and for implementation.

Reply:
I'm also not seeing a direct test for this, if it is implied by the remove_hs tests, please note that covers json truncation as well.

Reply:
I have a comment. Yes, the test was indeed implied in `test_get_mol_remove_hs()` (I noticed the problem while writing that test); I have now added an explicit test in `Code/GraphMol/MolInterchange/molinterchange_catch.cpp`.